package.json

```diff
@@ -13,5 +16,8 @@
       "type": "MIT",
       "url": "http://www.opensource.org/licenses/mit-license.php"
     }
-  ]
+  ],
+  "engines": {
+    "node": ">=6.4.0"
+  }
```

Comment:
Let's split the difference. We are starting to roll out webpack-defaults which has amongst other things, a proper Babel configuration.

I'll get the update done this evening, you can rebase it in and have access to all the ES6+ you want.